Symfony/CS/Tests/BasicFileFixingTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests;
+
+use Symfony\CS\Fixer;
+
+/**
+ * Test the behavior of all fixers on some basic PHP files.
```

Comment:
I just does too much at same time. And previously it was IntegrationTest right ?